### PR TITLE
style : fonts, correction button play, police hover color, add variab…

### PR DESCRIPTION
**Pull request by FrancoisSalmon**
…le to color on aswers

### CODE REVIEW

**Review by Gonaline — DISMISSED** [review]
ok for me

**Review by Gonaline — APPROVED** [review]
ok for me

**Review by LuigiMenez — APPROVED** [review]
Ok for me

File: frontend/index.html [hunk 1]
```diff
@@ -3,6 +3,10 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/src/favicon.svg" />
+    <link
```
Comment (LoicBrassart) [inline frontend/index.html:6]: Indentation is off

**Comment by LoicBrassart** [comment]
Saw non Evil

**Review by LoicBrassart — APPROVED** [review]
Saw non Evil

**Pull request merged**